lib/editor/util/gtfs.js

```diff
@@ -178,7 +178,24 @@ export const generateProps = (component: string, editorState: any): any => {
   }
 }
 
-export function getEntityBounds (entity: any, stops?: Array<GtfsStop>, offset: number = 0.005): ?latLngBounds {
+export function getEntityBounds (entity: any, stops?: Array<GtfsStop>, locations?: Array<GtfsLocation>, offset: number = 0.005): ?latLngBounds {
+  function findLocationCoodinates (coordinateArray) {
+    const lats = []
+    const lons = []
+
+    coordinateArray.forEach(point => {
+      // $FlowFixMe flow doesn't understand this but it works
+      lats.push(point.geometry_pt_lat || point.stop_lat)
+      // $FlowFixMe flow doesn't understand this but it works
+      lons.push(point.geometry_pt_lon || point.stop_lon)
+    })
```

Comment:
This might be cleaner as a map that maps into two arrays, although I'll leave it up to the second reviewer to decide (especially since this is existing code)